ui-components/typesoftoolbarbuttons.md

```diff
@@ -97,20 +97,18 @@ The callback given to `editor.formatter.formatChanged` is a function that takes
-| fetch | (success: (menu) => void) => void  | required | Function that takes a callback which must be passed the list of options for the button's dropdown. |
-| onAction | (api) => void | required | Function invoked when the basic button section is clicked. |
-| onItemAction | (api, value) => void | required | Function invoked when a dropdown list option is clicked. |
-| onSetup | (api) => (api) => void | optional | default: () => () => {} - Function invoked when the button is rendered. |
+| Name         | Value                             | Target component  | Requirement | Description                                                                                                                                                                                                               |
+|--------------|-----------------------------------|-------------------|-------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| text         | string                            | Primary button    | optional    | Text displayed if no icon is found.                                                                                                                                                                                       |
+| icon         | string                            | Primary button    | optional    | Name of the icon to be displayed. Must correspond to an icon in the [icon pack]({{site.baseurl}}/advanced/editor-icon-identifiers/) or an icon in a [custom icon pack]({{site.baseurl}}/advanced/creating-an-icon-pack/). |
+| onAction     | (api) => void                     | Primary button    | required    | Function invoked when the basic button section is clicked.                                                                                                                                                                |
+| select       | (value: string) => boolean        | Choice menu items | optional    | default: false - Function run on each option to determine if it should be highlighted as active.                                                                                                                          |
+| columns      | string                            | Drop-down menu    | optional    | default: 1 - Number of columns for the list of options.                                                                                                                                                                   |
+| fetch        | (success: (menu) => void) => void | Drop-down menu    | required    | Function that takes a callback which must be passed the list of options for the button's dropdown.                                                                                                                        |
```

Comment:
This... doesn't seem clear but I can't think of anything better

Reply:
Maybe 
> A callback function that is passed a list of choice menu items for the dropdown menu.

Reply:
May have been fixed by https://github.com/tinymce/tinymce-docs/pull/1573/commits/67539b3f222703c4de3af5e10a6c9848d82ea612